doc/releases/release-notes-2.1.rst

```diff
@@ -42,6 +65,18 @@ Architectures
   * Added support for CONFIG_DYNAMIC_INTERRUPTS (native_posix
     & nrf52_bsim)
 
+* RISC-V:
+
+  * Add support to boot multicore system
+
+* x86:
+
+  * Add basic ACPI and non-trivial memory map support
+  * Add SMP support (64-bit mode only)
+  * Add Z_ARCH_THREAD_STACK_MEMBER()
+  * Inline direct ISR functions
+  * Migrate Intel64 to z_arch_switch()
```

Comment:
lets remove that. I will give it another go after this is merged...

Reply:
ok